script.js

```diff
@@ -1391,9 +1391,49 @@ function exportVideo(blob) {
   vid.currentTime = Number.MAX_SAFE_INTEGER;
 }
 
+function uploadSupportedType(files) {
+  let supportedType = [
```

Comment:
Let's just use the `ext_map` on line 1192. We can make that global and then use `if (!(extension in ext_map)) {`.